apps/events.md

```diff
@@ -14,9 +14,11 @@ These events are available:
 | Event | Type | Description |
 | - | - |  - |
 | `pageview` | string | Is triggered when the customer opens or reloads a page. Informs about the path of the page. |
-| `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product. |
+| `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product as well as the current state of the cart. |
 | `category` | immutable.js object | Is triggered when the customer selects a category. Informs about the respective category and related products.|
+| `cart` | object | Is triggered when the customer accesses the cart. Informs about the current state of the cart, e.g. included items. |
```

Comment:
Is `object` correct?

Reply:
Yes, it is

Reply:
Sorry, I just noticed there is also `immutable.js object`.
- For `product` event: product and cart should be `immutable.js object`
- For `cart:add` and `cart` event: cart should be `object`